src/interp/interp-wasi.cc

```diff
@@ -35,6 +35,7 @@
 
 #include "uvwasi.h"
 
+#include <cstdio>
```

Comment:
Is this needed?

Reply:
was for testing and finding a memory-leak culprit. is removed. Thanks for the notice ._.